raianalysis/README.md

```diff
@@ -0,0 +1,16 @@
+# Responsible AI Analysis SDK for Python
+
+### This package has been tested with Python 3.5, 3.6 and 3.7
+
+The responsible AI analysis sdk enables users to analyze their machine learning models in one API. Users will be able to analyze errors, explain the most important features, analyze fairness, compute counterfactuals and run causal analysis in a single API.
+
+Highlights of the package include:
+
+- add_fairness allows users to run fairlearn to assess model fairness
```

Comment:
nit: add backquotes around function names to make them look like code

Reply:
oh cool idea, done